src/ui/components/DashKit/utils.ts

```diff
@@ -121,25 +130,44 @@ export function adjustWidgetLayout({
         return;
     }
 
-    const rootNodeTopPosition = node.getBoundingClientRect().top;
     const scrollableNodeTopPosition = scrollableNode.getBoundingClientRect().top;
-
     const scrollableNodeTopOffsetFromRoot = scrollableNodeTopPosition - rootNodeTopPosition;
     const belowLyingNodesHeight = collectBelowLyingNodesHeight(scrollableNode, node, 0);
 
-    const {scrollHeight} = scrollableNode;
-
-    // If widget has horizontal scroll, then we must add scrollBar height to fullContentHeight.
-    // We consider that horizontal scrollBar is equal to vertical scrollBar,
-    // so we could calculate horizontal scrollBar size.
-
-    let scrollBar = 0;
-    if (scrollableNode.scrollWidth > scrollableNode.clientWidth) {
-        scrollBar = (scrollableNode as HTMLElement).offsetWidth - scrollableNode.clientWidth;
+    // Calculationg scrollHeight without scroll and scrollbar width
+    let scrollBar = getScrollbarWidth(scrollableNode);
+    let scrollHeight = scrollableNode.scrollHeight;
+
+    if (scrollBar > 0) {
+        // If scrollBar is presented and there is scalable content
+        // calculating height without scroll
+        const overflowY = scrollableNode.style.overflowY;
+        scrollableNode.style.overflowY = 'hidden';
```

Comment:
Why do you use `somenode.style` in this full block? If there is an element with inline style it would return style properly. But if you have css style `overflow: auto;` on element this would return empty string. So how it works without `getComputedStyle`?
I can't get it. Only If we check situation when we added this style manually later after all calculations

Reply:
Rewrote this part on 'style' attribute with `!important`